python/migrate.py

```diff
@@ -0,0 +1,195 @@
+import json
+from typing import Any, Dict, List
+
+import mgp
+import mysql.connector as mysql_connector
+import oracledb
+import pyodbc
+
+I_COLUMN_NAME = 0
+
+
+@mgp.read_proc
+def mysql(
+    config: Any,
+    table_or_sql: str,
+    params: mgp.Nullable[mgp.Any] = None,
+) -> mgp.Record(row=mgp.Map):
+    """
+    With migrate.mysql you can access MySQL and execute queries. The result table is converted into a stream,
+    and returned rows can be used to create or create graph structures.
+
+    :param config: Connection configuration parameters (as in mysql.connector.connect), path to the JSON file
```

Comment:
Can you give an example of a call to this function with path and path and extension list here in the comment?